lms/static/js/instructor_dashboard/util.js

```diff
@@ -379,7 +379,7 @@
                 url: listEndpoint,
                 success: function(data) {
                     if (data.tasks.length) {
-                        ths.createTaskListTable(ths.$table_running_tasks, data.tasks);
+                        createTaskListTable(ths.$table_running_tasks, data.tasks); // eslint-disable-line no-undef
```

Comment:
We shouldn't ever be using undefined variables unless they're globals brought in from other files in non-AMD code. This should be `this.createTaskListTable`, and we shouldn't use this eslint ignore anywhere (if it's declared in a different file, add it to `/* globals` at the top).